msgraph-connector-test/pom.xml

```diff
@@ -156,4 +156,13 @@
       </plugin>
     </plugins>
   </build>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.glassfish.jersey.core</groupId>
+        <artifactId>jersey-client</artifactId>
+        <version>2.45</version><!-- fix: streamline with ivy-core -->
+      </dependency>
+    </dependencies>
```

Comment:
gonna have to release a web-tester:12.0.1 due to this version incompatibility ... the core is ahead and as jersey is used at runtime to bootstrap rest-mock endpoints ... it fails with NoSuchMethod exception.